### DISCUSSION TITLE
Make Cache Clearing Accessible in 'Free up Space

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#3965] [Enhancements]
Users currently have to go into advanced settings to clear the cache, which can be a hassle, especially when storage runs low, and the option isn’t easy to find. 

It’d be much easier if cache management were part of the “Free up space” section for quick access.

**Comment by NeverRIEght** [comment]
Could agree with your point. Actually it would be nice of Ente team to unify the options in settings.
For example it could be a directory named "Memory" or "Storage" and it will contain everything related to memory management, such as:
- Deduplicate images
- Free up space
- Clear Caches
- Clear Logs (?)
